woof-code/rootfs-petbuilds/connman-gtk/petbuild

```diff
@@ -9,5 +9,5 @@ build() {
     patch -p1 < ../icons.patch
     meson --buildtype=release --prefix=/usr -Duse_status_icon=false build
     ninja -C build install
-    sed -e 's/^Categories=.*/Categories=Network;/' -e 's~^Icon=.*~Icon=network_connect~' -i /usr/share/applications/connman-gtk.desktop
-}
\ No newline at end of file
+    rm -rf /usr/share/applications
```

Comment:
Why this change?

Reply:
Sorry, mistake while resolving git merge.